### PR TITLE
[CI] Update Single file Nightly Tests

### PR DESCRIPTION
**Pull request by DN6**
# What does this PR do?
Update our single file nightly tests to account for runway repo being removed. Updates the repo ids and single file urls to use Lykon's dreamshaper 8

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: tests/single_file/single_file_testing_utils.py [hunk 1]
```diff
@@ -98,8 +99,8 @@ def test_single_file_components_local_files_only(self, pipe=None, single_file_pi
         pipe = pipe or self.pipeline_class.from_pretrained(self.repo_id, safety_checker=None)
 
         with tempfile.TemporaryDirectory() as tmpdir:
-            ckpt_filename = self.ckpt_path.split("/")[-1]
-            local_ckpt_path = download_single_file_checkpoint(self.repo_id, ckpt_filename, tmpdir)
+            repo_id, weight_name = _extract_repo_id_and_weights_name(self.ckpt_path)
```
Comment (sayakpaul) [inline tests/single_file/single_file_testing_utils.py:102]: Feels like an overkill to me in using a method for just doing a splitting and indexing. 

Or is it more nuanced than that?

**Review by sayakpaul — APPROVED** [review]
Just a single command. Should we maintain a clone of the Lykon checkpoints on https://huggingface.co/hf-internal-testing/? If Lykon checkpoints get removed for whatever reasons, it'd be a nightmarish experience to go through tests again.

File: tests/single_file/single_file_testing_utils.py (hunk 1 above)
Comment (DN6) [reply tests/single_file/single_file_testing_utils.py:102]: Extracts the repo id and weight name from valid HF hosted repo urls 
https://github.com/huggingface/diffusers/blob/8ba90aa706a733f45d83508a5b221da3c59fe4cd/src/diffusers/loaders/single_file_utils.py#L293

**Comment by DN6** [comment]
> Just a single command. Should we maintain a clone of the Lykon checkpoints on https://huggingface.co/hf-internal-testing/? If Lykon checkpoints get removed for whatever reasons, it'd be a nightmarish experience to go through tests again.

I think it's fine for now. It matches the fallback repo for what we have for SD1.5

**Pull request merged**